Gemfile.lock

```diff
@@ -0,0 +1,80 @@
+PATH
```

Comment:
Gemfile.lock shouldn't be part of the repo. It's not an app that is deployed somewhere.